client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.js

```diff
@@ -0,0 +1,15 @@
+angular.module('managerApp')
+  .config(($stateProvider) => {
+    $stateProvider
+      .state('telecom.pack.xdsl.access-modem-exchange', {
+        url: '/comfortExchange',
+        views: {
+          'accessView@telecom.pack.xdsl': {
```

Comment:
Create a component instead please

Reply:
Why? It's a new view like for notifications, resiliation...
I'm not agree with you with this feedback.

Reply:
Why ? It's a view. Could you please more explicit?

Reply:
I think what @FredericEspiau meant is instead of declaring your view with a template and a controller, you should create a component (https://docs.angularjs.org/guide/component) and declare this view like this : 
```suggestion
          'accessView@telecom.pack.xdsl': {
           component: 'xdslAccessComfortExchange'
          }
```